packages/components/psammead-image/src/index.jsx

```diff
@@ -32,13 +32,39 @@ const StyledImg = styled.img`
   width: 100%;
 `;
 
+const getMimeType = srcset => {
+  if (!srcset) return null;
+
+  const firstSrcsetUrl = srcset.split(',')[0].split(' ')[0];
+  const urlFileExtension = firstSrcsetUrl.split('.').pop();
+
+  switch (urlFileExtension) {
+    case 'webp':
+      return 'image/webp';
+    case 'jpg':
+    case 'jpeg':
+      return 'image/jpeg';
+    case 'png':
+      return 'image/png';
+    case 'gif':
+      return 'image/gif';
+    default:
+      return null;
```

Comment:
don't we support the `raw` extension as well? Are there any other extensions missing here?

Reply:
`raw` is used as an iChef recipe, so something like `https://ichef.bbci.co.uk/news/raw/myfile.jpg`. It doesn't support the '.raw' file extension as it were.